server.js

```diff
@@ -1,38 +1,34 @@
 // TASK 1: Setting up server
 
 
-
 const http = require('http');
 
```

Comment:
There should be a newline before the server.listen() call for better readability.